hail/python/hailtop/gear/deploy_config.py

```diff
@@ -42,7 +42,7 @@ def domain(self, service):
         if self._location == 'k8s':
             return f'{service}.{ns}'
         if self._location == 'gce':
-            return 'hail.internal'
+            return 'internal.hail'
```

Comment:
So `internal.hail` should be used when you're connecting to a non-default namespace.  Do you want:

```
  if ns == 'default':
    return f'{service}.hail'
  return 'internal.hail'
```

To be analogous with the (external) gateway?

Reply:
If we do that, then I need to add DNS entries for `batch2.hail` etc. And they'll all point to the same URL that's the static one I provisioned.

Reply:
Yeah.  Right now we're only connecting to batch2, so we should have entries for batch2.hail and internal.hail.  Does google support wildcard DNS entries?  So you can just point *.hail to the internal IP.  That's what I did with the external DNS pointing to the external static IP.

 > same URL

You mean the same IP?  Yeah, they should all go to the same place.

Reply:
That way I don't need to edit the DNS when we add/remove services, just the router rules.

Reply:
Didn't realize the wildcard was possible.